src/cr/cube/matrix.py

```diff
@@ -472,38 +589,87 @@ def rows(self):
 
     @lazyproperty
     def table_base(self):
+        """np.int64 count of actual respondents who answered both questions.
```

Comment:
Oh, now I think I know... both refers to the cross-section from _both_ dimensions? Ok, I'll give the PR another pass, and rethink the docstrings from before.